feinstaub/feinstaub/urls.py

```diff
@@ -1,8 +1,15 @@
+
 from django.conf.urls import include, url
 from django.contrib import admin
 from django.views.generic.base import RedirectView
 from rest_framework.authtoken.views import obtain_auth_token
 
+from django.conf.urls.static import static
+
+
+from django.conf import settings
+
+
```

Comment:
Let's reduce the number of blank lines between each of these. Check out for reference https://www.python.org/dev/peps/pep-0008/#blank-lines